index.ts

```diff
@@ -130,6 +131,9 @@ export {Type} from './api_types';
 export type {TypedPackFormula} from './api';
 export type {Formula} from './api';
 export {UserVisibleError} from './api';
+export {ParameterValidationError} from './api';
+export {ParameterError} from './api';
+export {LegacyDefaultMetadataReturnType} from './api';
```

Comment:
Do we need to export this in the package index? Generally preferable not to export intermediate internal types.